### PR TITLE
fix: add override for `/app/installations/` response

### PR DESCRIPTION
**Pull request by Jman420**
Add override for /app/installations/ response for account property to be an allOf list instead of anyOf

Resolves https://github.com/octokit/openapi-types.ts/issues/305

---

### Before the change?

- The response for /app/installations returned an array of objects who's account property was an intersection of the simple-user & enterprise schemas.  This hides all of the non-intersecting properties making them inaccessible.
- See https://github.com/github/rest-api-description/issues/2497 for more details.

### After the change?

- Add an override which changes the anyOf descriptor to allOf for the account property.
- See https://github.com/github/rest-api-description/issues/2497 for more details.

### Pull request checklist

- [ n/a ] Tests for the changes have been added (for bug fixes / features)
- [ n/a ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [ ] Yes
- [ ] No
- [x] Maybe? - The override technically changes the type of the response's account property but the existing type is broken as it does not include all of the necessary fields.

---

### CODE REVIEW

**Comment by Jman420** [comment]
I think the Dry-run is failing due to the build script not handling parsing the version from the filename correctly.  I think the regex currently only supports a single digit for the two parts of the version and the download script has downloaded a file for v3.10.  Version 3.10 is also not included in the SUPPORTED_GHES_OPERATIONS constant.

Should I include fixes to those issue in this PR?

**Comment by kfcampbell** [comment]
@Jman420 unfortunately that's affecting all the PRs at the moment. If you do fix that, would you mind using a separate PR?

File: scripts/overrides/index.mjs [hunk 1]
```diff
@@ -106,6 +106,14 @@ export default function overrides(file, schema) {
   );
 
   if (isDeferenced(file)) {
+    // The /app/installations/ endpoint have bad usage of `anyof` in the response body schema
```
Comment (wolfy1339) [inline scripts/overrides/index.mjs:109]: Can you add a link to the issue you opened for this fix
```suggestion
    // The `/app/installations/` endpoint has bad usage of `anyof` in the response body schema
    // See https://github.com/octokit/openapi-types.ts/issues/305
```

File: scripts/overrides/index.mjs [hunk 2]
```diff
@@ -134,6 +142,14 @@ export default function overrides(file, schema) {
       "./repos-compare-commits.deref.json",
     );
   } else {
+    // The /app/installations/ endpoint have bad usage of `anyof` in the response body schema
```
Comment (wolfy1339) [inline scripts/overrides/index.mjs:145]:
```suggestion
    // The `/app/installations/` endpoint has bad usage of `anyof` in the response body schema
    // See https://github.com/octokit/openapi-types.ts/issues/305
```

**Review by wolfy1339 — APPROVED** [review]

**Review by wolfy1339 — APPROVED** [review]

**Pull request merged**